pkg/agent/status_manager.go

```diff
@@ -128,6 +128,8 @@ func (mgr *Manager) updateClusterStatus(ctx context.Context, namespace, clusterI
 			return false, nil
 		}
 
+		// don't update conditions in agent
+		status.Conditions = mgr.managedCluster.Status.Conditions
```

Comment:
What if the agent comes back to life again ? Does legacy "NOT READY" condition still persist ?

Reply:
yes, if we don't add this line in agent, we will see a status with empty conditions when agent comes back to life and 'Ready' condition will be added soon by hub.

Reply:
What if the hub crashes?

Reply:
so maybe agent is responsible for "READY" "NOT READY" conditions, hub is responsible for "UNKNOWN" conditions.